tests/Hedgehog.Tests/SeedTests.fs

```diff
@@ -25,7 +24,5 @@ let seedTests = testList "Seed tests" [
           (0x76442b62dddf926cUL,  8100917074368564322UL, 12297829382473034421UL) ] <| fun (x, value, gamma) ->
 
         { Value = value
-          Gamma = gamma }
-        =! Seed.from x
-
+          Gamma = gamma } =! Seed.from x
```

Comment:
```fsharp
let actual =
  { Value = value
    Gamma = gamma }

actual =! Seed.from x
```

Reply:
:+1: (Space [isn't needed](https://blog.ploeh.dk/2013/06/24/a-heuristic-for-formatting-code-according-to-the-aaa-pattern/) there.)